src/App.tsx

```diff
@@ -1,14 +1,50 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { getTodos } from './api';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+
+import { StatusFilter } from './modules/StatusFilter';
+
+import { getFilteredTodos } from './utils/getFilteredTodos';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const [status, setStatus] = useState<StatusFilter>(StatusFilter.ALL);
+  const [query, setQuery] = useState('');
+
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  const filteredTodos = getFilteredTodos(todos, { query, status });
+
+  function handleCloseModul() {
+    setSelectedTodo(null);
+  }
+
+  function getTodosData() {
+    setIsLoading(true);
+
+    getTodos()
+      .then(setTodos)
+      // eslint-disable-next-line no-console
+      .catch(error => console.log(error))
```

Comment:
You are directly logging the error to the console. It would be better to handle the error in a more user-friendly way. For example, you could set an error state and display a message to the user when an error occurs.